test/e2e/resize_nodes.go

```diff
@@ -341,7 +428,8 @@ var _ = Describe("Nodes", func() {
 			}
 		})
 
-		testName = "should survive network partition."
+		// TODO: this test has nothing to do with resizing nodes so it should be moved elsewhere.
+		testName = "should survive unresponsive node."
```

Comment:
I added a TODO in the code. I'll put this task on my list (at the end, unfortunately)